src/Analyzer/FileVisitor.php

```diff
@@ -41,17 +41,29 @@ public function enterNode(Node $node): void
          * @see FileVisitorTest::test_should_returns_all_dependencies
          */
         if ($node instanceof Node\Expr\StaticCall && method_exists($node->class, 'toString')) {
+            if ('self' === $node->class->toString()) {
```

Comment:
This code portion is repeated inside the next three if conditions:
```
            if ('self' === $node->class->toString()) {
                return;
            }
            $this->classDescriptionBuilder
                ->addDependency(new ClassDependency($node->class->toString(), $node->getLine()));
```

Could be a good idea refactor it ?